questionnaires/webops.md

```diff
@@ -20,11 +21,23 @@
 - What is ports and what are they used for
 - How many ports
 - What is a TCP router?
+- How is TCP different from UDP?
+- What is congestion control?
```

Comment:
@jotaen i need this material / questions to be effective and practical for teaching beginners in web ops or intermediate web devs. I'd see congestion control as advanced topic, agree? i think i will create a separate question sets for beginners / advanced.

Reply:
To be fair, I happened to see that you created this repo last week, and I was just in the mood to dump some questions that came to my mind spontaneously. So feel free to take or leave what you like 😄 

But regarding your question, I’d agree that congestion control is more advanced.